beluga/test/beluga/sensor/data/test_ndt_cell.cpp

```diff
@@ -0,0 +1,73 @@
+// Copyright 2023 Ekumen, Inc.
```

Comment:
@serraramiro1 nit: `2023` -> `2024`, here and elsewhere.

Reply:
Done